components/client-attestation-mgt/org.wso2.carbon.identity.client.attestation.mgt/src/main/java/org/wso2/carbon/identity/client/attestation/mgt/services/ClientAttestationServiceImpl.java

```diff
@@ -92,49 +89,40 @@ public ClientAttestationContext validateAttestation(String attestationObject,
                 LOG.debug("App :" + serviceProvider.getApplicationResourceId() + " in tenant : " + tenantDomain +
                         " is not subscribed to Client Attestation Service.");
             }
-            clientAttestationContext.setAttestationEnabled(false);
+            clientAttestationContext = new ClientAttestationContext(false);
```

Comment:
don't we need to add this
```
        clientAttestationContext.setApplicationResourceId(applicationResourceId);
        clientAttestationContext.setTenantDomain(tenantDomain);
```

Reply:
The applicationResourceId and tenant domain is added at the end of method after the if condition - https://github.com/wso2/carbon-identity-framework/pull/5642/files#diff-08802b242cf7c44399e7f440ee3af001f1b7bdea33c76a52338707d139d775a9R123-R125